model/ftn/ww3_shel.ftn

```diff
@@ -1258,7 +1258,7 @@
           END IF
           IF (WORDS(7) .EQ. 'T') THEN
             CALL NEXTLN ( COMSTR , NDSI , NDSEN )
-            READ (NDSI,*,END=2001,ERR=2002) FLDRST
+            READ (NDSI,'(A)',END=2001,ERR=2002) FLDRST
```

Comment:
Why was this changed? Was the list directed read not working correctly for FLDRST?

Reply:
No, this was not impacting any existing regtests. It is now checked in ww3_tp2.14, in particular in the OASACM4 and OASACM5 tests (the first writes the restart file, the second reads it).

Reply:
Thanks Juan.